src/test/java/plugins/FindBugsPluginTest.java

```diff
@@ -177,6 +177,7 @@ public void should_set_result_to_unstable_if_warning_found() {
      * Builds a pipeline project and checks if it is unstable.
      */
     @Test
+    @WithPlugins("workflow-aggregator")
```

Comment:
Is it still ok to use workflow-aggregator? Didn't @jglick suggest to not use it anymore and make the dependencies explicit? Or is this guideline just for plug-ins and not ATH?

Reply:
Hi, yeah you are right - I totally forgot! I will change it for something with less dependencies.